flink-ml-lib/src/main/java/org/apache/flink/ml/classification/knn/DenseMatrix.java

```diff
@@ -0,0 +1,161 @@
+package org.apache.flink.ml.classification.knn;
+
+import java.io.Serializable;
+
+/**
+ * Knn DenseMatrix stores dense matrix data and provides some methods to operate on the matrix it
+ * represents. This data structure helps knn to accelerate distance calculation.
+ */
+public class DenseMatrix implements Serializable {
```

Comment:
Instead of putting `DenseMatrix`, distance classes in this package, I think it could be better to place them in common packages like `distance` or `linalg`.

Reply:
OK, I agree with you. I will move it later.

Reply:
done, thanks for your suggestion.